ui/redux/reducers/settings.js

```diff
@@ -46,6 +46,7 @@ const defaultState = {
     [SETTINGS.HIDE_BALANCE]: false,
     [SETTINGS.OS_NOTIFICATIONS_ENABLED]: true,
     [SETTINGS.AUTOMATIC_DARK_MODE_ENABLED]: false,
+    [SETTINGS.CLOCK_24H]: false,
```

Comment:
I have no idea what this does, but the other settings were here so I added this one too.

Reply:
This is the default value for this setting, which looks good